src/common/read-maybe-existed-file.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+const fs = require("fs");
+
+/**
+ * @param {string} filename
+ * @returns {Promise<null | string>}
+ */
+function readMaybeExistedFile(filename) {
```

Comment:
read file if exists?

I found this name a bit weird but I don't have any good suggestions besides `safeReadFile` or something like that.

Reply:
getFileContentOrNull?